core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -870,6 +870,7 @@ class TransactionsTest extends IntegrationTestHarness {
     }
   }
 
+  @Flaky("KAFKA-18306")
```

Comment:
I have the PR open for this -- https://github.com/apache/kafka/pull/18264, so do we still want to mark it?
As an aside, what is the process for removing the tag? I removed the others in my PR but didn't know if there was a time we wanted to analyze the new results before removing.

Reply:
I think our general strategy should be to aggressive in marking things as flaky, and cautious about un-marking them. There's also no harm in marking something flaky and immediately removing the tag in a subsequent PR (assuming we are confident in the fix).

Reply:
Would you mind taking a look at my PR fixing it? 😉